### DISCUSSION TITLE
Rinkeby network

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#1557] [Q&A] [answered]
Hi! I am in the lessson 1. I can not add Rinkeby network to my metamask account. And it asks me to add it in custom.

![image](https://user-images.githubusercontent.com/108202008/175796843-1def29ee-1bba-4202-8f6f-2af7352a2cc5.png)

**Comment by cromewar** [comment, marked as answer]
Hello @slklk 
Go to settings -> advanced -> Show/HIde test networks and activate it. 
This should solve your problem ;)
